src/atoms/Flex/__tests__/Flex.web.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
More tests please

Reply:
added more tests

Reply:
@krishnaacc123 we need to test if inline styles are being applied because that was one edge case. You can take reference from native tests for Flex.

Reply:
@krishnaacc123  we need to test if inline styles are being applied because that was one edge case. You can take reference from native tests for Flex.